stdlib/2and3/codecs.pyi

```diff
@@ -2,6 +2,7 @@
 # https://docs.python.org/2/library/codecs.html and https://docs.python.org/3/library/codecs.html
```

Comment:
Honestly the attribution comment does not belong here, nor does the value judgment.